sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1Test.java

```diff
@@ -272,23 +272,19 @@ public void testReadDisplayDataWithGqlQuery() {
   @Test
   public void testSourcePrimitiveDisplayData() {
     DisplayDataEvaluator evaluator = DisplayDataEvaluator.create();
-    PTransform<PBegin, PCollection<Entity>> read = DatastoreIO.v1().read().withProjectId(
-        "myProject").withQuery(Query.newBuilder().build());
-
+    int numSplits = 98;
+    PTransform<PBegin, PCollection<Entity>> read =
+        DatastoreIO.v1().read()
+            .withProjectId(PROJECT_ID)
+            .withQuery(Query.newBuilder().build())
+            .withNumQuerySplits(numSplits);
+
+    String assertMessage = "DatastoreIO read should include the '%s' in its primitive display data";
     Set<DisplayData> displayData = evaluator.displayDataForPrimitiveSourceTransforms(read);
-    assertThat("DatastoreIO read should include the project in its primitive display data",
-        displayData, hasItem(hasDisplayItem("projectId")));
-  }
-
-  @Test
-  public void testSourcePrimitiveDisplayDataWithGqlQuery() {
```

Comment:
Removed this as it wasn't adding much value, the paths for `withQuery` and `withGqlQuery` are the same for the display data under test.